src/crawlers/cheerio_crawler.js

```diff
@@ -468,9 +452,6 @@ class CheerioCrawler {
                 return dom && !isXml && $.html({ decodeEntities: false });
             },
             get json() {
-                // TODO (JC): The naming here is quite unfortunate. The returned thing here is not a JSON,
```

Comment:
We know, but it's also unfortunate to have parsed JSON saved in `body` when parsed HTML is in `$` and string is in `body`. It's also suboptimal to have unparsed JSON in `json`, because it would be the same as `body` and would require an extra `JSON.parse()` call in user land. Finally, naming it `object` or `objectFromJson` felt really weird.

So we settled with unfortunate `json` and now it's used in 800+ scrapers by a client, so I'm not really in favour of making a breaking change to this.

Reply:
Cool, I just need to write my heart out :)